algo/src/main/java/org/neo4j/graphalgo/metaPathComputationProcs/ComputeAllMetaPathsSchemaFullProc.java

```diff
@@ -35,19 +40,22 @@
 
         final ComputeAllMetaPathsSchemaFullResult.Builder builder = ComputeAllMetaPathsSchemaFullResult.builder();
 
-        final ComputeAllMetaPathsSchemaFull algo = new ComputeAllMetaPathsSchemaFull(length, api);
-        HashSet<String> metaPaths;
+        org.neo4j.graphdb.Result queryResult;
+        try (Transaction tx = api.beginTx()) {
+            queryResult = api.execute("CALL algo.GetSchema();");
+            tx.success();
+        }
+        Map<String, Object> row = queryResult.next();
+        Gson gson = new Gson();
+        ArrayList<HashSet<Pair>> schema = gson.fromJson((String) row.get("schema"), new TypeToken<ArrayList<HashSet<Pair>>>(){}.getType());
+        HashMap<Integer, Integer> reversedLabelDictionary = gson.fromJson((String) row.get("reverseLabelDictionary"),  new TypeToken<HashMap<Integer, Integer>>(){}.getType());
+
+        final ComputeAllMetaPathsSchemaFull algo = new ComputeAllMetaPathsSchemaFull(length, schema, reversedLabelDictionary);
+
         ComputeAllMetaPathsSchemaFull.Result result = algo.compute();
```

Comment:
Import instead of giving the whole path

Reply:
Dann haben wir nen ambiguous reference error für Result